cypress/fixtures/details-fixture.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
{data: album {info}}

Reply:
The following commit address fixes this configuration issue:
"Reconfigure fixture object" 5df2c62